ur_kinematics fails to build on indigo

It looks like a missing include file.

```
00:08:02 In file included from /tmp/binarydeb/ros-indigo-ur-kinematics-1.1.6/src/ur_moveit_plugin.cpp:88:0:
00:08:02 /tmp/binarydeb/ros-indigo-ur-kinematics-1.1.6/include/ur_kinematics/ur_moveit_plugin.h:97:74: fatal error: moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp: No such file or directory
00:08:02  #include <moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp>
00:08:02     
```

http://build.ros.org/view/Ibin_uS64/job/Ibin_uS64__ur_kinematics__ubuntu_saucy_amd64__binary/41/console

--- a/ur_kinematics/CMakeLists.txt
+++ b/ur_kinematics/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ur_kinematics)
 
-find_package(catkin REQUIRED COMPONENTS roscpp geometry_msgs moveit_core
+find_package(catkin REQUIRED COMPONENTS roscpp geometry_msgs moveit_core moveit_kinematics
   moveit_ros_planning pluginlib tf_conversions)
 
 find_package(Boost REQUIRED COMPONENTS system)
@@ -9,7 +9,7 @@ find_package(Boost REQUIRED COMPONENTS system)
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES ur3_kin ur5_kin ur10_kin ur3_moveit_plugin ur5_moveit_plugin ur10_moveit_plugin
-  CATKIN_DEPENDS roscpp geometry_msgs moveit_core moveit_ros_planning
+  CATKIN_DEPENDS roscpp geometry_msgs moveit_core moveit_kinematics moveit_ros_planning
     pluginlib tf_conversions
   DEPENDS boost
 )

--- a/ur_kinematics/package.xml
+++ b/ur_kinematics/package.xml
@@ -18,6 +18,7 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend>moveit_core</build_depend>
+  <build_depend>moveit_kinematics</build_depend>
   <build_depend>moveit_ros_planning</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>geometry_msgs</build_depend>
@@ -26,6 +27,7 @@
   <build_depend>boost</build_depend>
 
   <run_depend>moveit_core</run_depend>
+  <run_depend>moveit_kinematics</run_depend>
   <run_depend>moveit_ros_planning</run_depend>
   <run_depend>roscpp</run_depend>
   <run_depend>geometry_msgs</run_depend>

